### PR TITLE
feat: add triple term names

### PR DESCRIPTION
**Pull request by jeswr**
Adds a `TripleTermName` util type.

### CODE REVIEW

**Review by rubensworks — APPROVED** [review]

File: query/common.d.ts [hunk 1]
```diff
@@ -4,10 +4,15 @@
...
+/**
+ * Helper union type for triple term names.
+ */
+export type TripleTermName = 'subject' | 'predicate' | 'object';
+
 /**
  * Helper union type for quad term names.
  */
-export type QuadTermName = 'subject' | 'predicate' | 'object' | 'graph';
+export type QuadTermName = TripleTermName | 'graph';
```
Comment (tpluscode) [inline query/common.d.ts:15]: I never realised we **did** have `QuadTermName` 😅

File: README.md [hunk 2]
```diff
@@ -26,6 +26,14 @@ import type { DatasetCore } from '@rdfjs/types/dataset'
 import type { Stream } from '@rdfjs/types/stream'
 ``````
 
+## Helper Terms
```
Comment (tpluscode) [inline README.md:29]:
```suggestion
## Additional Types
```

**Review by tpluscode — COMMENTED** [review]
I only propose to change the docs header. Otherwise, looks good

File: query/common.d.ts (hunk 1 above)
Comment (jeswr) [reply query/common.d.ts:15]: Neither did I until I tried to add the same thing lol

**Review by tpluscode — APPROVED** [review]

**Comment by jeswr** [comment]
> I only propose to change the docs header. Otherwise, looks good

kewl - I don't have permission to merge, can you do that @tpluscode

**Comment by tpluscode** [comment]
There you go 🚀

**Pull request merged**